.github/workflows/deploy-docs.yaml

```diff
@@ -10,6 +10,14 @@ jobs:
       contents: write
     runs-on: ubuntu-latest
     steps:
+    - uses: actions/setup-go@41dfa10bad2bb2ae585af6ee5bb4d7d973ad74ed # 5.1.0
+    - run: |
+        pushd nodeadm
+        GOOS=js GOARCH=wasm go build -o nodeadm.wasm cmd/nodeadm-wasm/main.go
```

Comment:
Can you move this into a target of `nodeadm/Makefile` so it's easier to run locally and is covered by the PR checks?

Reply:
added `make wasm` target and `nodeadm-build-wasm` workflow 👍